src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketScmFormValidationDelegate.java

```diff
@@ -89,7 +89,7 @@ public FormValidation doCheckRepositoryName(String serverId, String credentialsI
             return FormValidation.ok(); // There will be an error in the credentials field
         }
         if (isEmpty(repositoryName)) {
-            return FormValidation.error("Please specify a repository name.");
+            return FormValidation.error("Required");
```

Comment:
Same reasoning as before. Because the user sees the label, help text, and validation all at once I think it is better to simplify here and to leave this as is.